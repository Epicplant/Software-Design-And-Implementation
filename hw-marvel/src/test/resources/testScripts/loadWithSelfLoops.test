# Tests that every node doesn't have an edge to itself even when there are intentional self-loops

# Loads a new graph from a csv file
LoadGraph newGraph selfLoopHeroGraph.csv

# Lists all of the children of A
ListChildren newGraph A