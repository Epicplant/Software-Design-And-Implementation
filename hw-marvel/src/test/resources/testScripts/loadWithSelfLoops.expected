# Tests that every node doesn't have an edge to itself even when there are intentional self-loops

# Loads a new graph from a csv file
loaded graph newGraph

# Lists all of the children of A
the children of A in newGraph are: B(1)